src/library_openal.js

```diff
@@ -1765,6 +1765,7 @@ var LibraryOpenAL = {
     };
     var onSuccess = function(mediaStream) {
       newCapture.mediaStreamSourceNode = newCapture.audioCtx.createMediaStreamSource(mediaStream);
+      mediaStreamTrack = mediaStream;
```

Comment:
this looks like it assigns to the global scope - there should probably be a `var ` in the proper scope for it.

Reply:
Now I've declared mediaStreamTrack at $AL.